src/main/resources/interface-specification.yml

```diff
@@ -6325,6 +6325,161 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /clients:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    get:
+      tags:
+        - clients
+      summary: retrieves a list of clients
+      description: retrieves a list of clients
+      operationId: getClients
+      parameters:
```

Comment:
let's use only the parameters requested in the story, and hide the others in the implementation

Reply:
+1

Reply:
Apologize me, but I have some questions about this issue:
consumerId is mandatory how can achieve it without pass as a parameter?
purposeId is not mandatory so I can pass None
kind the same thing above


Can you enlighten me on the situation?

Reply:
for the `consumerId` you can use the `organizationId` in the jwt.
I spoke with the front, let's leave the `kind` parameter, which is actually useful